site/docs/java-api-quickstart.md

```diff
@@ -41,6 +41,9 @@ import org.apache.iceberg.catalog.TableIdentifier;
 
 TableIdentifier name = TableIdentifier.of("logging", "logs");
 Table table = catalog.createTable(name, schema, spec);
+
+// or to load an existing table, use the following line
+// Table table = catalog.loadTable(name);
```

Comment:
I think we should add a section for this instead of adding this block in multiple places. I'm happy to do that if you want to keep it separate from this PR.

Reply:
Ah yes I did this because having a new section for loading table would duplicate the code here. It'd be easier for us to have a different PR (or a PR against my fork) if you have better idea. Thanks!